src/shared/components/adminTabs/FlashRound/FlashRoundAdmin.tsx

```diff
@@ -128,6 +160,7 @@ const FlashRoundAdmin = () => {
             })}
             {game && (
                 <>
+                    <small>Suma punctelor trebuie sa fie 100!</small>
```

Comment:
We should not give the users such restrictions

Reply:
Good catch :) am uitat